src/libraries/Microsoft.PowerFx.Core/Public/Values/BlankValue.cs

```diff
@@ -28,5 +28,17 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            if (Type is BlankType)
+            {
+                return Type.DefaultExpressionValue();
+            }
+            else
```

Comment:
can we remove this special check?

Reply:
Without it we get a weird type: **Blank() ({?})**

Reply:
Wouldn't it just be:
If(false, Blank())

Does that have a different type than:
Blank()?

Reply:
The following test cases illustrate this scenario:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/19541904/193842766-bde317d8-d2ee-499e-97d2-4fb18ebdfd8c.png">

Reply:
Can we add that as a comment? 
// if(False, Blank()) actually generates an unknown type, not blank type.